tests/rptest/tests/rbac_test.py

```diff
@@ -640,13 +640,16 @@ def _license_nag_is_set(self):
         )
 
     @cluster(num_nodes=1)
+    @ok_to_fail_fips  # See NOTE below
```

Comment:
question/nit: does `@ok_to_fail_fips` still run this test? I'm wondering if skipping this test in fips mode would be more appropriate. Is there a `@skip_fips` decorator perhaps? (Don't worry about it if it doesn't exist / it's not trivial to add).